rx_paparazzo/src/main/java/com/fuck_boilerplate/rx_paparazzo/RxPaparazzo.java

```diff
@@ -157,6 +166,14 @@ public BuilderImages(T ui) {
         }
 
         /**
+         * Call it when crop features is required.
+         */
+        public BuilderImages<T> cropFeatures(UCropFeature feature) {
```

Comment:
This method should be named `crop`, being that way an overloaded version of existing `crop`.